pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -100,11 +100,16 @@ func (z *zedrouter) attachNTPServersToPortConfigs(portConfigs []nireconciler.Por
 
 		ntpServers := make([]net.IP, 0, len(ntpServerDomainsOrIPs))
 		for _, ntpServer := range ntpServerDomainsOrIPs {
+			if len(strings.TrimSpace(ntpServer)) == 0 {
+				continue
+			}
+
 			ip := net.ParseIP(ntpServer)
```

Comment:
Since you use TrimSpace about, don't you want to use the trimmed string here as well?